### PR TITLE
MC-1146 Maps dropdown related to the selected WAN replication

### PR DESCRIPTION
**Pull request by Sodik**

### CODE REVIEW

File: docs/modules/monitor-imdg/pages/monitor-wan-replication.adoc [hunk 1]
```diff
@@ -118,7 +118,7 @@ and navigate to the **Sync** tab.
 
 image:ROOT:WanSyncTab.png[WAN Sync Tab]
 
-Select *Endpoint*(replication config), *Target*(publisher), and one or many *Maps*
+Select *Endpoint*(replication config), *Target*(publisher), and one or many *Maps*(related to the selected replication)
```
Comment (Serdaro) [inline docs/modules/monitor-imdg/pages/monitor-wan-replication.adoc:121]: ... one or more ...

**Review by Serdaro — APPROVED** [review]
Minor edit suggestion/

File: docs/modules/monitor-imdg/pages/monitor-wan-replication.adoc (hunk 1 above)
Comment (Sodik) [reply docs/modules/monitor-imdg/pages/monitor-wan-replication.adoc]: done

[the remark this answers is not in the document]
Comment (Sodik) [reply docs/modules/monitor-imdg/pages/monitor-wan-replication.adoc]: done

**Pull request merged**